### PR TITLE
Few fixes and updates

### PR DESCRIPTION
**Pull request by pmb59**
This PR includes:
- few edits in the README
- updates run sorting-hat script template with new variables
- changes branch of atlas-analysis (move away from peach API) - see related [PR](https://github.com/ebi-gene-expression-group/atlas-analysis/pull/9)
- remove constraint on 'gsea.tsv' and 'gsea_list.tsv' files as required input for `rule copy_experiment_from_analysis_to_atlas_exps`,  as they could be correctly removed by `rule check_differential_gsea` for certain studies (when they are empty)
- require analysis_methods files for rule `atlas_experiment_summary`

### CODE REVIEW

File: envs/quantile.yaml
Comment (anilthanki) [inline envs/quantile.yaml]: Any particular reason for removing version.. ? it always picks latest version of not specified and in future if may have conflict

Comment (pmb59) [reply envs/quantile.yaml]: @anilthanki  I found errors in analysis of microarray data. I am not sure at the moment if this errors are data-specific or because the new conda envs got some conflict between few dependencies. I'll make sure to clarify before merging this PR

File: Snakefile
Comment (anilthanki) [inline Snakefile]: Just for the readability of the code 

```suggestion
        if experiment_type in ['rnaseq_mrna_differential', 'proteomics_differential', 'microarray_1colour_mrna_differential' , 'microarray_2colour_mrna_differential', 'microarray_1colour_microrna_differential']:
```

File: .gitmodules
Comment (irisdianauy) [inline .gitmodules]: This can be brought back to develop_codon, as feature/fix_postprocessing has been merged to develop_codon. Good to merge after this.

**Review by irisdianauy — APPROVED** [review]
Once comment has been addressed, this looks good to merge.

File: Snakefile
Comment (pmb59) [reply Snakefile]: Thanks @anilthanki that makes sense

**Pull request merged**
